source/reference/method/db.collection.watch.txt

```diff
@@ -115,13 +115,15 @@ Definition
       * - ``batchSize``
    
         - int
-        - Optional. Specifies the maximum number of change events to return in each
-          batch of the response from the MongoDB cluster. 
+        - Optional. The maximum number of documents within each batch returned in a change stream. 
+          By default, :method:`~db.collection.watch()` has an initial batch size of ``101`` documents and a maximum size 
+          of 16 mebibytes (MiB) for each subsequent batch. This option can enforce a smaller 
+          limit than 16 MiB, but not a larger one. If you set ``batchSize`` to a limit that
+          results in batches larger than 16 MiB, this option has no effect.
```

Comment:
[behavior question]

Can we clarify what we mean by "no effect" here? I take it that means it uses the default.